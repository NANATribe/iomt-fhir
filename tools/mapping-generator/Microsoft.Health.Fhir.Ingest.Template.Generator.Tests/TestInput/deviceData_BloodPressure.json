{
  "type": "Blood Pressure",
  "time": "2022-07-01T12:01:00Z",
  "device": "TestDeviceId",
  "patient": "TestPatientId",
  "encounter": "TestEncounter",
  "correlation": "TestCorrelation",
  "values": {
    "diastolic": 79,
    "systolic": 119
  }
}
