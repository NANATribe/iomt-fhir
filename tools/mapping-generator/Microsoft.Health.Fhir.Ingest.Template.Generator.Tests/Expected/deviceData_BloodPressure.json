{
  "templateType": "CalculatedContent",
  "template": {
    "typeMatchExpression": "$..[?(@.type=='Blood Pressure')]",
    "deviceIdExpression": "$.device",
    "patientIdExpression": "$.patient",
    "encounterIdExpression": "$.encounter",
    "timestampExpression": "$.time",
    "correlationIdExpression": "$.correlation",
    "values": [
      {
        "valueName": "diastolic",
        "valueExpression": "$.values.diastolic"
      },
      {
        "valueName": "systolic",
        "valueExpression": "$.values.systolic"
      }
    ],
    "typeName": "Blood Pressure"
  }
}